kong/cmd/utils/prefix_handler.lua

```diff
@@ -168,6 +195,19 @@ local function compile_conf(kong_config, conf_template)
     end
   end
 
+  if type(kong_config.lua_ssl_trusted_certificate) == "table" then
```

Comment:
Since `lua_ssl_trusted_certificate = { typ = "array" },`, wouldn't this be true always?

Reply:
It can be `nil` on tests